NikitaNechaiev/HW6/HW1.py

```diff
@@ -0,0 +1,19 @@
+#even numbers that are divisible by 2,
```

Comment:
good job
look at this code
x1 = []
x2 = []
x3 = []

for x in range(1, 11):
    if x % 2 == 0:
        #print(x, 'is even multiple of 2')
        x1.append(x)
    elif x % 3 == 0:
        #print(x, 'is an odd multiple of 3')
        x2.append(x)    
    else:
        #print(x, 'not divisible by 2 and 3')
        x3.append(x)
print(x1)        
print(x2)
print(x3)